crates/bevy_core_pipeline/src/core_3d/main_pass_3d_node.rs

```diff
@@ -14,6 +14,27 @@ use bevy_render::{
 #[cfg(feature = "trace")]
 use bevy_utils::tracing::info_span;
 
+/// Add a `PrepassSettings` component to a view to perform a depth and/or normal prepass.
+/// These textures are useful for reducing overdraw in the main pass, and screen-space effects.
+#[derive(Clone, Component)]
+pub struct PrepassSettings {
+    /// If true then depth values will be copied to a separate texture available to the main pass.
+    pub output_depth: bool,
+    /// If true then vertex world normals will be copied to a separate texture available to the main pass
```

Comment:
Missing a period at the end of the documentation for output_vertex.

Reply:
```suggestion
    /// If true then vertex world normals will be copied to a separate texture available to the main pass.
```